[2024-12-19T05:45:18.041+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T05:45:18.049+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2024-11-01T00:00:00+00:00 [queued]>
[2024-12-19T05:45:18.053+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2024-11-01T00:00:00+00:00 [queued]>
[2024-12-19T05:45:18.054+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T05:45:18.061+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2024-11-01 00:00:00+00:00
[2024-12-19T05:45:18.064+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5134) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T05:45:18.065+0000] {standard_task_runner.py:72} INFO - Started process 5143 to run task
[2024-12-19T05:45:18.065+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2024-11-01T00:00:00+00:00', '--job-id', '1195', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmprb1gr92s']
[2024-12-19T05:45:18.066+0000] {standard_task_runner.py:105} INFO - Job 1195: Subtask first_task
[2024-12-19T05:45:18.090+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2024-11-01T00:00:00+00:00 [running]> on host 13c961a72918
[2024-12-19T05:45:18.127+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-01T00:00:00+00:00'
[2024-12-19T05:45:18.127+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T05:45:18.136+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T05:45:18.136+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T05:45:18.140+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T05:45:18.143+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T05:45:18.144+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T05:45:18.154+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T05:45:18.154+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2024-11-01T00:00:00+00:00, execution_date=20241101T000000, start_date=20241219T054518, end_date=20241219T054518
[2024-12-19T05:45:18.163+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T05:45:18.169+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T05:45:18.170+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
